### PR TITLE
audit event decoupling

### PR DESCRIPTION
**Pull request by shimonewman**
## What does this PR do?

Decouple Audit events from Events in order to get more human-readable events.

### CODE REVIEW

**Review by eldadfux — APPROVED** [review]

File: app/controllers/api/functions.php [hunk 1]
```diff
@@ -1240,6 +1246,7 @@
     ->desc('Retry Build')
     ->label('scope', 'functions.write')
     ->label('event', 'functions.[functionId].deployments.[deploymentId].update')
+    ->label('audits.event', 'build.create')
```
Comment (christyjacob4) [inline app/controllers/api/functions.php:1249]: sholdnn't this be `deployment.update` ?

Comment (eldadfux) [reply app/controllers/api/functions.php:1249]: It should.

File: src/Appwrite/Event/Audit.php [hunk 2]
```diff
@@ -108,6 +108,29 @@ public function getIP(): string
...
+    /**
+     * Returns the  audit event description.
+     *
+     * @return string
+     */
+    public function getEvent(): string
+    {
+        return $this->event;
+    }
+
```
Comment (TorstenDittmann) [inline src/Appwrite/Event/Audit.php:133]: This is already part of the parent class 👍🏻

**Review by TorstenDittmann — CHANGES_REQUESTED** [review]

File: src/Appwrite/Event/Audit.php (hunk 2 above)
Comment (TorstenDittmann) [reply src/Appwrite/Event/Audit.php:133]: So we don't need it here.

Comment (TorstenDittmann) [reply src/Appwrite/Event/Audit.php:133]: https://github.com/appwrite/appwrite/blob/342c4a3253816b9fb72b6ede26a617e31e0c362e/src/Appwrite/Event/Event.php#L81-L101

Comment (shimonewman) [reply src/Appwrite/Event/Audit.php]: Cool

File: app/controllers/api/functions.php (hunk 1 above)
Comment (shimonewman) [reply app/controllers/api/functions.php]: Ok

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**